internal/controller/runtime.go

```diff
@@ -133,8 +133,13 @@ loop:
 
 		// GPU
 		if gpu != nil {
+			// Training requires more memory. For now use double the amount of GPUs
+			if runtime == RuntimeNotebook || runtime == RuntimeTrainer {
+				gpuCount = gpuCount * 2
```

Comment:
because training requires more memory. It's somehwat documented here: https://huggingface.co/docs/transformers/perf_train_gpu_one#anatomy-of-models-memory